data_steward/bq_utils.py

```diff
@@ -297,6 +297,23 @@ def delete_table(table_id, dataset_id=None):
     return delete_job.execute(num_retries=bq_consts.BQ_DEFAULT_RETRY_COUNT)
 
 
+def delete_dataset(project_id, dataset_id):
```

Comment:
Because we are trying to move toward using the client library, I would remove this from the `bq_utils` module.  This will encourage using the new bq client library utils module.

Reply:
I agree we need to use the new module. However this involves changing all current instances of bq_utils to use bq. Do you think it would be better in a separate PR? Maybe something that looks at deprecating bq_utils.py completely.

Reply:
I was referring to the `delete_dataset` function this PR creates in `bq_utils` only.  We do plan to deprecate `bq_utils` completely, in the future.  Right now, I don't think we should be adding functions to `bq_utils`.  Especially if nothing is using that function.  Your script is specifically using the function in the new wrapper module.  So, this is a function that no one is using?.?.?.

Reply:
Ah, my bad! Sorry I was thinking it already existed and was in use.